build.sbt

```diff
@@ -429,6 +452,7 @@ lazy val `sdk-exporter` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     sdk,
     `sdk-exporter-common`,
     `sdk-exporter-metrics`,
+    `sdk-exporter-prometheus`,
```

Comment:
I wonder whether we should distribute the Prometheus exporter in the exporter 'bundle'. The `http4s-ember-server` and `http4s-dsl` dependencies would be added to the classpath, even if users rely exclusively on the OTLP exporters. 

Alternatively, we can mark `http4s-ember-server` and `http4s-dsl` dependencies as `provided`. But to be honest, I don't know how viable this idea is.

Reply:
Fair point. Indeed, it's unlikely that anyone will need to use both OTLP and Prometheus exporters in one application at the same time. So, removed.